src/overlays/actors/ovl_Obj_Tokei_Tobira/z_obj_tokei_tobira.h

```diff
@@ -2,16 +2,27 @@
 #define Z_OBJ_TOKEI_TOBIRA_H
 
 #include "global.h"
+#define OBJTOKEITOBIRA_GET_TYPE(thisx) ((thisx)->params & 1)
```

Comment:
```suggestion
#include "global.h"

#define OBJTOKEITOBIRA_GET_TYPE(thisx) ((thisx)->params & 1)
```
Small nit but can you add a newline here.

Reply:
Bump

Reply:
Would like to bump on this again and see if you could address this small little nit.